Documentation/operations/upgrade.rst

```diff
@@ -347,6 +347,10 @@ New Options
   ``tunnel=disabled``, now deprecated.
 * ``tunnel-protocol``: This option allows setting the tunneling protocol, in place
   of e.g., ``tunnel=vxlan``.
+* ``tls-relay-client-ca-files``: This option allow providing certificate authority
+  key and cert in Hubble Relay to authenticate its clients with mTLS. When provided,
+  Hubble Relay will enforce mTLS authentication on its clients (i.e. Hubble CLI
+  client won't be able to connect to Hubble Relay using ``--tls-allow-insecure``).
```

Comment:
Use clear, plain language, specific is better than general, use present tense, no Latin abbreviations
```suggestion
* ``tls-relay-client-ca-files``: This option lets you provide a certificate authority (CA)
  key and cert in Hubble Relay to authenticate Hubble Relay's clients with mTLS. When you provide a CA key and cert,
  Hubble Relay enforces mTLS authentication on its clients (for example, Hubble CLI
  client can't connect to Hubble Relay using ``--tls-allow-insecure``).
```